guides/source/caching_with_rails.md

```diff
@@ -42,105 +50,98 @@ INFO: Action Caching has been removed from Rails 4. See the [actionpack-action_c
 
 ### Fragment Caching
 
-Life would be perfect if we could get away with caching the entire contents of a page or action and serving it out to the world. Unfortunately, dynamic web applications usually build pages with a variety of components not all of which have the same caching characteristics. In order to address such a dynamically created page where different parts of the page need to be cached and expired differently, Rails provides a mechanism called Fragment Caching.
+Dynamic web applications usually build pages with a variety of components not
+all of which have the same caching characteristics. When different parts of the
+page need to be cached and expired separately you can use Fragment Caching.
 
 Fragment Caching allows a fragment of view logic to be wrapped in a cache block and served out of the cache store when the next request comes in.
 
-As an example, if you wanted to show all the orders placed on your website in real time and didn't want to cache that part of the page, but did want to cache the part of the page which lists all products available, you could use this piece of code:
+For example, if you wanted to cache each product on a page, you could use this
+code:
 
 ```html+erb
-<% Order.find_recent.each do |o| %>
-  <%= o.buyer.name %> bought <%= o.product.name %>
-<% end %>
-
-<% cache do %>
-  All available products:
-  <% Product.all.each do |p| %>
-    <%= link_to p.name, product_url(p) %>
+<% @products.each do |product| %>
+  <% cache product do %>
+    <%= render product %>
   <% end %>
 <% end %>
 ```
 
-The cache block in our example will bind to the action that called it and is written out to the same place as the Action Cache, which means that if you want to cache multiple fragments per action, you should provide an `action_suffix` to the cache call:
+When your application receives its first request to this page Rails will write
+a new cache to a unique key. A key looks something like this:
 
-```html+erb
-<% cache(action: 'recent', action_suffix: 'all_products') do %>
-  All available products:
+```html
+views/products/1-201505056193031061005000/bea67108094918eeba42cd4a6e786901
 ```
 
-and you can expire it using the `expire_fragment` method, like so:
+The number in the middle is the `product_id` followed by the timestamp value in
+the `updated_at` cell of the product table. Rails uses the timestamp value to
```

Comment:
Well, I guess you can simply say "the `updated_at` value of the product record".

At the very least it should be either "product**s** table" or "product record". Then, if you refer to the table you should use "column" but that doesn't really make sense here as the cache key is generated for a single record. So it's better to speak about "value" or "attribute" here instead of "column" (I'm not sure "cell" is widely used in this domain).